### PR TITLE
Add test reg virt-who host using UUID

### PR DESCRIPTION
**Pull request by swadeley**
A test for registering  a virt-who host to Satellite using UUID not name.

### CODE REVIEW

**Comment by swadeley** [comment]
robottelo]$ pytest --show-capture=no tests/foreman/virtwho/test_cli_virtwho.py -k uuid
========================== test session starts ==================================
platform linux -- Python 3.7.5, pytest-4.6.3, py-1.8.1, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /sat6/virtwho/robottelo
plugins: cov-2.8.1, services-1.3.1, mock-1.10.4, forked-1.1.3, xdist-1.31.0
collecting ... 2020-01-31 17:04:46 - conftest - DEBUG - Collected 10 test cases
2020-01-31 17:04:51 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f0e65e53850
2020-01-31 17:04:51 - robottelo.ssh - INFO - Connected to [ent-02-vm-07.lab.eng.nay.redhat.com]
2020-01-31 17:04:51 - robottelo.ssh - INFO - >>> rpm -q satellite
2020-01-31 17:04:52 - robottelo.ssh - INFO - <<< stdout
satellite-6.7.0-5.beta.el7sat.noarch

2020-01-31 17:04:52 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f0e65e53850
2020-01-31 17:04:52 - robottelo.host_info - DEBUG - Host Satellite version: 6.7
2020-01-31 17:04:52 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 10 items / 9 deselected / 1 selected                                                                   

tests/foreman/virtwho/test_cli_virtwho.py .                                                                [100%]

====================== 1 passed, 9 deselected in 327.40 seconds ===============

**Comment by swadeley** [comment]
Thank you @mirekdlugosz , fixed and force pushed.

File: tests/foreman/virtwho/test_cli_virtwho.py [hunk 1]
```diff
@@ -412,3 +412,49 @@ def test_positive_post_hypervisors(self):
                 )
             else:
                 self.assertTrue(result.status_code == 200)
+
+    @tier2
+    def test_positive_register_virt_who_host_with_uuid(self):
```
Comment (eko999) [inline tests/foreman/virtwho/test_cli_virtwho.py]: @swadeley, actually, we have a case to cover  `hypervisor_id=uuid` already,  please refer to:
`def test_positive_hypervisor_id_option`

Comment (swadeley) [reply tests/foreman/virtwho/test_cli_virtwho.py]: OK, will check and test it.

Comment (JacobCallahan) [reply tests/foreman/virtwho/test_cli_virtwho.py]: @swadeley any luck with that check? If you believe that test covers, would you mind closing this pr?

Comment (swadeley) [reply tests/foreman/virtwho/test_cli_virtwho.py]: No luck, dev unresponsive. Closing.

**Pull request closed without merging**